packages/smooth_ui_library/lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -29,56 +23,73 @@ class SmoothAlertDialog extends StatefulWidget {
   final List<SmoothSimpleButton> actions;
 
   @override
-  _SmoothAlertDialogState createState() => _SmoothAlertDialogState();
+  _SmoothAlertDialogState createState() => _SmoothAlertDialogState(
+        title: title,
+        close: close,
+        height: height,
+        body: body,
+        actions: actions,
+      );
 }
 
 class _SmoothAlertDialogState extends State<SmoothAlertDialog> {
+  _SmoothAlertDialogState(
+      {this.title, this.close, this.height, this.body, this.actions});
+
+  final String title;
+  final bool close;
+  final double height;
+  final Widget body;
+  final List<SmoothSimpleButton> actions;
```

Comment:
Again these attributes are unnecessary, you access the ones from the Widget using `widget.attribute`